### PR TITLE
Transformations API 2.0: replace ngraph headers and namespaces with openvino

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
 - *Using API 2.0 headers and namespaces instead of ngraph in transformation source files*

### Tickets:
 - *CVS-111373*

### CODE REVIEW

**Review by evkotov — APPROVED** [review]

**Review by mateusztabaka — APPROVED** [review]

File: src/common/transformations/src/itt.hpp [hunk 1]
```diff
@@ -13,22 +13,22 @@
 #include <openvino/itt.hpp>
 #include <openvino/cc/ngraph/itt.hpp>
 
-namespace ngraph {
+namespace ov {
 namespace pass {
 namespace itt {
 namespace domains {
     OV_ITT_DOMAIN(IETransform);
```
Comment (ilyachur) [inline src/common/transformations/src/itt.hpp:20]:
```suggestion
    OV_ITT_DOMAIN(Transform);
```

File: src/common/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp [hunk 2]
```diff
@@ -5,9 +5,9 @@
 #include "transformations/common_optimizations/add_fake_quantize_fusion.hpp"
 
 #include <memory>
-#include <ngraph/pattern/op/wrap_type.hpp>
-#include <ngraph/rt_info.hpp>
-#include <ngraph/validation_util.hpp>
+#include <openvino/core/rt_info.hpp>
+#include <openvino/core/validation_util.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
```
Comment (ilyachur) [inline src/common/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp:10]: Please use "" instead of <>

Comment (itikhono) [reply src/common/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp]: done

File: src/common/transformations/src/itt.hpp (hunk 1 above)
Comment (itikhono) [reply src/common/transformations/src/itt.hpp]: done

File: src/common/transformations/src/precomp.hpp [hunk 3]
```diff
@@ -4,8 +4,8 @@
 
 #pragma once
 
-#include <ngraph/ngraph.hpp>
-#include <ngraph/ops.hpp>
+#include <ngraph/ngraph.hpp"
```
[the remark this answers is not in the document]
Comment (itikhono) [reply src/common/transformations/src/precomp.hpp]: fixed

**Review by ilyachur — APPROVED** [review]

**Pull request merged**